### PR TITLE
Fix ability to turn a third card

### PR DESCRIPTION
**Pull request by DevCas1**

### CODE REVIEW

File: MemoryGame-1B/Src/MemoryGrid.cs [hunk 1]
```diff
@@ -257,6 +257,10 @@ private void CardClick(object sender, MouseButtonEventArgs e)
         {
             if (CardData.Length - CardData.Cast<CardData>().Count(x => x.Turned && x.FoundPair) == 2) return;
```
Comment (JipSterk) [inline MemoryGame-1B/Src/MemoryGrid.cs]: Try without this line should also work

File: MemoryGame-1B/Src/Managers/GameManager.cs [hunk 2]
```diff
@@ -55,6 +55,11 @@ public static class GameManager
         /// </summary>
         public static int ScorePlayer2 { get; set; }
 
+        /// <summary>
+        /// Keeps track of how many cards have been turned in one turn
+        /// </summary>
+        public static int CardsTurned { get; set; }
```
Comment (JipSterk) [inline MemoryGame-1B/Src/Managers/GameManager.cs]: Could this be private?

File: MemoryGame-1B/Src/MemoryGrid.cs (hunk 1 above)
Comment (DevCas1) [reply MemoryGame-1B/Src/MemoryGrid.cs]: Works without that line!

File: MemoryGame-1B/Src/Managers/GameManager.cs (hunk 2 above)
Comment (DevCas1) [reply MemoryGame-1B/Src/Managers/GameManager.cs]: Sadly, no. MemoryGrid requires it

**Pull request merged**